content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,172 @@
+============
+Work Entries
```

Comment:
I'll go ahead and make it lower case, since it CAN be the official label (caps) or the generic meaning of the phrase, so I'll make it lower case (I can change it back if Zac says).